repository/kafka/operator/params.yaml

```diff
@@ -94,6 +109,46 @@ DELETE_RECORDS_PURGATORY_PURGE_INTERVAL_REQUESTS:
   default: "1"
   description: "The purge interval (in number of requests) of the delete records request purgatory"
 
+KERBEROS_ENABLED:
+  description: "Enable kerberos authentication."
+  default: "false"
+  displayName: "Kerberos Enabled"
+
+KERBEROS_ENABLED_FOR_ZOOKEEPER:
+  description: "Enable Kerberos authentication for communication with Apache Zookeeper."
+  default: "false"
+  displayName: "Zookeeper Kerberos Enabled"
+
+KERBEROS_PRIMARY:
+  description: "The Kerberos primary used by Kafka tasks."
+  default: "kafka"
+  displayName: "Kerberos Primary"
+
+KERBEROS_KEYTAB_SECRET:
+  description: "The name of the Kubernetes secret storing the keytab."
+  default: "base64-kafka-keytab-secret"
+  displayName: "Kerberos Keytab Secret"
+
+KERBEROS_REALM:
+  description: "The Kerberos realm used to render the principal of Kafka broker pods."
+  default: "LOCAL"
+  displayName: "Kerberos Realm"
+
+KERBEROS_KDC_HOSTNAME:
+  description: "The name or address of a host running a KDC for the realm."
+  default: "kdc-service.kudo-kafka.svc.cluster.local"
```

Comment:
let's remove the usage of namespace `kudo-kafka` from default values 
I know we have used it extensively in our docs to explain how KUDO Kafka features work. But lets use headless services as default values as we do for `zookeeper` for example

Reply:
BTW its totally fine how it's used in the documentation 🙂